assets/javascripts/discourse/templates/verification-intents/show.hbs

```diff
@@ -0,0 +1,36 @@
+{{#if model.processing}}
+  <p style="text-align:center;">
+    Your verification currently in progress. Please wait...
+  </p>
+  <div class="spinner"></div>
+{{/if}}
+{{#if model.requiresAction}}
+  <p style="text-align:center;">
+    Your verification has been failed.
+  </p>
+  <p style="text-align:center;">
+    <strong>Reason: {{model.error}}</strong>
+  </p>
+  <p style="text-align:center;">
+    Please click the link below and repeat verification process.
+  </p>
+  <a href={{model.verification_url}} style="text-align: center;display: block;margin: 0 auto;">
+    Verify
+  </a>
+{{/if}}
+{{#if model.canceled}}
+  <p style="text-align:center;">
+    Your verification has been cenceled.
+  </p>
+  <p style="text-align:center;">
+    Please contact with us to solve this issue.
+  </p>
+{{/if}}
+{{#if model.succeeded}}
+  <p style="text-align:center;">
+    Your verification has been completed.
```

Comment:
we should use i18n

Reply:
This text will be changed in the separate story when we will get the confirmation from the customer about design(https://www.pivotaltracker.com/story/show/173941827)